setup.cfg

```diff
@@ -40,6 +40,7 @@ install_requires =
   pandas>=0.23.0
   astropy>=3.2
   parfive[ftp]>=1.1.0
+  scikit-learn>=0.23.1
```

Comment:
This should be moved into its own extra dependency (as below) and the imports moved into the functions with an import check.

Reply:
Would the changes I have made suffice?